### ISSUE TITLE
Add Javascript for <b>` resume.html ` to show dynamic data pushed by user form

### ISSUE THREAD

**Issue by iamrishirb** [devclub-biet-jhansi/Online-ResumeBuilder#2]
The user form is where the main landing page is ` index.html `

User will enter his/ her details in this form to create the Resume -> after entering the details the page is redirected to ` resume.html ` as form's action. Right now, this page holds a basic template showing how actual data will be displayed after a user fills a form. The HTML markups are coded with full care using relevant field ID's and classes matching with form fields.

Some valid JavaScript / jQuery is required to make these fields display dynamic data given by user through **Resume Details Form**

**Comment by devclub-biet-jhansi** [comment]
Yes @iamrishirb I think this is a good thing to work on. Would you like working on it's JS part @avinash-chandravansi ?

**Comment by iamrishirb** [comment]
I'm intrested in solving this issue
